Sources/CoherentSwiftCore/Helpers/UserInputHelper.swift

```diff
@@ -26,7 +26,7 @@ public class UserInputHelper {
         let sourceFolder = userInput(
             with: "Provide the relative path of the source folder to be scanned during the report. Ensure this folder exists.",
             suggestion: "i.e.: ./MyApp/Sources/") { input -> Bool in
-            return Path(input).exists
+            return !Path.glob(input).isEmpty
```

Comment:
We actually want to ensure the folder exists.
In the user input isn't clear how to use multiple sources either, we should probably give an example in the `suggestion` above

Reply:
I'd say, to keep it simple for now, from cli it's only possible to specify 1 path e.g. `MyApp/*/Sources` and in `.yml` it can be more a list of paths. 
also it seems that GitHub Actions only support String variables which makes it hard to add a list of paths from it.